crates/bevy_app/src/plugin_group.rs

```diff
@@ -27,13 +31,22 @@ impl PluginGroup for PluginGroupBuilder {
 /// Provides a build ordering to ensure that [`Plugin`]s which produce/require a [`Resource`](bevy_ecs::system::Resource)
 /// are built before/after dependent/depending [`Plugin`]s. [`Plugin`]s inside the group
 /// can be disabled, enabled or reordered.
-#[derive(Default)]
 pub struct PluginGroupBuilder {
+    group_name: String,
```

Comment:
Since `PluginGroup::name` returns a `&'static str`, please use a `Cow<'static, str>` here. 

Alternatively remove the static lifetime requirement on the trait to make the trait easier to implement and keep this as a `String`.

Reply:
go for strings!

Reply:
go for strings!